atom/browser/native_window_mac.mm

```diff
@@ -1100,8 +1100,13 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
 void NativeWindowMac::SetOverlayIcon(const gfx::Image& overlay,
                                      const std::string& description) {}
 
-void NativeWindowMac::SetVisibleOnAllWorkspaces(bool visible) {
-  SetCollectionBehavior(visible, NSWindowCollectionBehaviorCanJoinAllSpaces);
+void NativeWindowMac::SetVisibleOnAllWorkspaces(bool visible,
+                                                bool visibleOnFullScreen) {
+  if (visible) {
```

Comment:
i'm not sure this boolean logic will work here: for something like 
```
if (a) doOneThing elseif (a && b) doOtherThing()```
```
there's no way to exercise the second codepath since you'll never hit a situation where the second conditional will be true and the first one won't be

Reply:
oops, fixed!